src/components/carousel/__tests__/index.spec.js

```diff
@@ -0,0 +1,83 @@
+import {map} from 'lodash';
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+import {Text, View} from 'react-native';
+import {Constants} from '../../../helpers';
+import Carousel from '../index';
+
+const numberOfPagesShown = 5;
+const eventData = {
+  nativeEvent: {
+    contentOffset: {
+      x: Constants.screenWidth
+    }
+  }
+};
+const onChangePageMock = jest.fn();
+const onScrollMock = jest.fn();
+const props = {
+  testID: 'carousel',
+  initialPage: 0,
+  pagingEnabled: true,
+  autoplay: false,
+  autoplayInterval: 4000,
+  horizontal: true,
```

Comment:
Please don't pass the carousel's default props, so we could test it as well

Reply:
Done